web/sites/default/config/filter.format.simplified_editor_with_media_embed.yml

```diff
@@ -16,7 +16,7 @@ filters:
     status: true
     weight: -50
     settings:
-      allowed_html: '<em> <strong> <cite> <blockquote cite> <code> <ul type> <li> <dl> <dt> <dd> <h2 id> <h3 id> <h4 id> <h5 id> <h6 id> <p class=""> <hr> <sup> <sub> <s> <img src alt data-entity-type data-entity-uuid> <pre> <i class=""> <drupal-entity data-* title alt data-entity-type data-entity-uuid data-entity-embed-display data-entity-embed-display-settings data-align data-caption data-embed-button data-langcode> <a href hreflang !href accesskey id rel target title class=" btn-cta btn-cta-outline"> <div class=" row col col-6 col-4-md col-3-md"> <ol start type class="code-section-list">'
+      allowed_html: '<em> <strong> <cite> <blockquote cite> <code> <ul type> <li> <dl> <dt> <dd> <h2 id> <h3 id> <h4 id> <h5 id> <h6 id> <p class=""> <hr> <sup> <sub> <s> <img src alt data-entity-type data-entity-uuid> <pre> <i class=""> <drupal-entity data-* title alt data-entity-type data-entity-uuid data-entity-embed-display data-entity-embed-display-settings data-align data-caption data-embed-button data-langcode> <a href hreflang !href accesskey id rel target title class=" btn-cta btn-cta-outline"> <div class=" row col col-6 col-4-md col-3-md"> <ol type start class="code-section-list">'
```

Comment:
@RichardDavies, I don't see a file change here. Do you? It's kinda weird. Maybe a line ending?

Reply:
Took me a minute to find it, but the ordering of the attributes of the <ol> tag at the very end changed (trivially).

Reply:
Oh... dang... I can't believe I didn't catch that. It is usually my speciality :)